### PR TITLE
Templatize the image repo url for the partner pod

### PR DESCRIPTION
**Pull request by jun-rh**

### CODE REVIEW

File: deploy-partner-pods.sh [hunk 1]
```diff
@@ -15,6 +14,6 @@ fi
 
 cat ./local-test-infra/local-partner-deployment.yaml | ./script/mo > ./temp/rendered-partner-template.yaml
 oc apply -f ./temp/rendered-partner-template.yaml
-rm ./temp/rendered-partner-template.yaml
+#rm ./temp/rendered-partner-template.yaml
```
Comment (edcdavid) [inline deploy-partner-pods.sh:17]: Did you mean to keep the rendered yaml?

**Review by hamadise — APPROVED** [review]

File: deploy-partner-pods.sh (hunk 1 above)
Comment (jun-rh) [reply deploy-partner-pods.sh:17]: no, that was for testing only.

Comment (jun-rh) [reply deploy-partner-pods.sh]: updated

**Comment by jun-rh** [comment]
Need to update README

**Review by edcdavid — APPROVED** [review]

**Comment by jun-rh** [comment]
> Need to update README

Pushed

**Pull request merged**